src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -364,7 +356,9 @@ const VirtualizedMessageListWithContext = <
         <div className={customClasses?.virtualizedMessageList || 'str-chat__virtual-list'}>
           <Virtuoso<UnknownType, VirtuosoContext<StreamChatGenerics>>
             atBottomStateChange={atBottomStateChange}
-            atBottomThreshold={200}
+            atBottomThreshold={100}
+            atTopStateChange={atTopStateChange}
```

Comment:
In regards to your comment in the PR description. Why don't we protect ourselves from integrators overriding this prop?

Reply:
With this change - we probably should, just wanted to see if there are any other drawbacks I could be missing and we'd be tossing this change altogether.